bellbot_action_server/conf/bellbot_voice.yaml

```diff
@@ -13,8 +13,8 @@ WaitingForGoal:
     start: "Waiting For Goal start"
     end: "Waiting For Goal end"
 Guiding:
-    start: "Guiding start"
-    end: "Guiding end"
+    start: "Guiding start to $(topic /current_node)"
```

Comment:
This doesn't make much sense... `"Guiding start to $(topic /current_node)"`... but i get the concept.